spec/system/admin/user_spec.rb

```diff
@@ -8,6 +8,34 @@
   context 'when a super admin is logged in' do
     include_context 'log in super admin'
 
+    describe 'active_scaffold list', :js do
+      before { visit url }
+
+      it 'is accessible' do
+        within('h2') do
+          expect(page).to have_content('Users')
+        end
+      end
+
+      it 'has expected columns' do
+        expected_headings = %I[
+          email-column
+          first_name-column
+          last_name-column
+          affiliate_names-column
+          default_affiliate-column
+          created_at-column
+          updated_at-column
+          approval_status-column
+          actions
+        ]
```

Comment:
Don't spend too much time refactoring (like: 5-10 mins max), but it would be preferable if we could verify the human-readable column headings as they appear on the page, i.e., "Email", "First Name", etc.)